From 1f3706d10812d70adefe32fe0d7d3a3ec25374f0 Mon Sep 17 00:00:00 2001
From: Hauke Mehrtens <hauke@hauke-m.de>
Date: Sun, 21 Nov 2021 00:02:57 +0100
Subject: Revert "iw: allow specifying CFLAGS/LIBS externally"

This reverts commit 1325244b77d56fd7a16d1e35fdae0efc151920b1.

The OpenWrt build system provides the CFLAGS and LIBS names from the 
package Makefile to overwrite them for libnl-tiny. This is not possible 
after this upstream change which we revert here any more

Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>
---
 Makefile | 20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

--- a/Makefile
+++ b/Makefile
@@ -46,30 +46,30 @@ NLLIBNAME = libnl-1
 endif
 
 ifeq ($(NL2FOUND),Y)
-override CFLAGS += -DCONFIG_LIBNL20
-override LIBS += -lnl-genl
+CFLAGS += -DCONFIG_LIBNL20
+LIBS += -lnl-genl
 NLLIBNAME = libnl-2.0
 endif
 
 ifeq ($(NL3xFOUND),Y)
 # libnl 3.2 might be found as 3.2 and 3.0
 NL3FOUND = N
-override CFLAGS += -DCONFIG_LIBNL30
-override LIBS += -lnl-genl-3
+CFLAGS += -DCONFIG_LIBNL30
+LIBS += -lnl-genl-3
 NLLIBNAME = libnl-3.0
 endif
 
 ifeq ($(NL3FOUND),Y)
-override CFLAGS += -DCONFIG_LIBNL30
-override LIBS += -lnl-genl
+CFLAGS += -DCONFIG_LIBNL30
+LIBS += -lnl-genl
 NLLIBNAME = libnl-3.0
 endif
 
 # nl-3.1 has a broken libnl-gnl-3.1.pc file
 # as show by pkg-config --debug --libs --cflags --exact-version=3.1 libnl-genl-3.1;echo $?
 ifeq ($(NL31FOUND),Y)
-override CFLAGS += -DCONFIG_LIBNL30
-override LIBS += -lnl-genl
+CFLAGS += -DCONFIG_LIBNL30
+LIBS += -lnl-genl
 NLLIBNAME = libnl-3.1
 endif
 
@@ -77,8 +77,8 @@ ifeq ($(NLLIBNAME),)
 $(error Cannot find development files for any supported version of libnl)
 endif
 
-override LIBS += $(shell $(PKG_CONFIG) --libs $(NLLIBNAME))
-override CFLAGS += $(shell $(PKG_CONFIG) --cflags $(NLLIBNAME))
+LIBS += $(shell $(PKG_CONFIG) --libs $(NLLIBNAME))
+CFLAGS += $(shell $(PKG_CONFIG) --cflags $(NLLIBNAME))
 endif # NO_PKG_CONFIG
 
 ifeq ($(V),1)
